DQM/SiStripMonitorClient/scripts/TkMap_script_automatic_DB.sh

```diff
@@ -188,22 +198,22 @@ do
     fi
 
 #Temporary fix to remove hidden ASCII characters
-    GLOBALTAG=`echo $GLOBALTAG | cut -c 9-${#GLOBALTAG}`
-#    GLOBALTAG=`sed -i 's/[\d128-\d255]//g' <<< "${GLOBALTAG}"`
-#    GLOBALTAG=`echo $GLOBALTAG | sed 's/[\d128-\d255]//'`
-#    echo `expr length $GLOBALTAG`
-
+#    GLOBALTAG=`echo $GLOBALTAG | cut -c 9-${#GLOBALTAG}`
+    
     echo " Creating the TrackerMap.... "
 
     detIdInfoFileName=`echo "file://TkDetIdInfo_Run${Run_numb}_${thisDataset}.root"`
 
-    #cmsRun ${CMSSW_BASE}/src/DQM/SiStripMonitorClient/test/SiStripDQM_OfflineTkMap_Template_cfg_DB.py print globalTag=${GLOBALTAG} runNumber=${Run_numb} dqmFile=${file_path}/$dqmFileName  # update GlobalTag
     cmsRun ${CMSSW_BASE}/src/DQM/SiStripMonitorClient/test/SiStripDQM_OfflineTkMap_Template_cfg_DB.py print globalTag=${GLOBALTAG} runNumber=${Run_numb} dqmFile=${file_path}/$dqmFileName  detIdInfoFile=${detIdInfoFileName} # update GlobalTag
 
 # rename bad module list file
 
      mv QTBadModules.log QualityTest_run${Run_numb}.txt
 
+# put color legend in the TrackerMap     
+    /usr/bin/python ${CMSSW_BASE}/src/DQM/SiStripMonitorClient/scripts/LegendToQT.py QTestAlarm.png FinalLegendTrans.png
```

Comment:
@davidlange6 I assume this has been tested in a different machine where the package PIL for python has been installed manually. Is there any problem in using the system python?